CHANGES.rst

```diff
@@ -3,6 +3,13 @@ Change Log
 
 prawcore follows `semantic versioning <http://semver.org/>`_.
 
+Unreleased
+------------------
```

Comment:
For RST this line should only be as long as the characters above it.

Reply:
Ok, I'll fix that